tgui/packages/tgui/interfaces/BlackMarketUplink.js

```diff
@@ -114,16 +114,16 @@ const ShipmentSelector = (props, context) => {
   });
   return (
     <Modal textAlign="center">
-      <Flex mb={1}>
+      <Stack mb={1}>
         {deliveryMethods.map(method => {
           if (method.name === 'LTSRBT' && !ltsrbt_built) {
             return null;
           }
           return (
-            <Flex.Item
+            <Stack.Item
               key={method.name}
               mx={1}
-              width="250px">
+              width="17.5rem">
```

Comment:
How does this look when you resize the UI? You may want to consider using grow instead.

Reply:
grow won't work here, it causes the elements to stack in a really strange manner. not sure if that's because it's using modal.
here's it, as is, window resized:
![1](https://user-images.githubusercontent.com/42397676/147145687-1ee3250d-b6ca-444f-a034-97ae5b3213a6.png)

Reply:
using grow:
![2](https://user-images.githubusercontent.com/42397676/147145854-d93965eb-8b07-413f-afc8-dcc18edb44b1.png)